web/src/types.ts

```diff
@@ -69,3 +69,10 @@ export type UpdateMessageEvent = {
 // TODO/typescript: Move the User and Stream placeholder
 // types to their appropriate modules.
 export type User = Record<string, never>;
+
+// Define the type Collision type as an export to its aprropiate module(debug.js)
```

Comment:
This type does not belong in `types.ts`, it belongs in `debug.ts`. Don’t put it in `types.ts` with a comment that it should be moved to `debug.ts` later. Instead, put it in `debug.ts` now.